app.ts

```diff
@@ -0,0 +1,75 @@
+interface Student {
+    address?: string;
+    age?: number;
+    avgGrade?: number;
+    city?: string;
+    email: string;
+    id?: number;
+    firstName: string;
+    lastName: string;
+  }
+  
+ 
+  
+  type Teams = "Olympaikos" | "Panathinaikow" | "AEK" | "Aris" | "PAOK";
+
+  //question 1
+     function sendStudentData1():Partial<Student>{
+      return {
+        avgGrade: 20,
+        id: 12345,
+        firstName: "Tasos",
+        lastName: "Matatakis"   
+      }
+    }
+
```

Comment:
Nice implementation of Partial, though the function does not send the data to the server